samples/bluetooth/iso_connected_benchmark/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  name: BLE ISO Connected
+  description: Bluetooth Low Energy ISO Connected Benchmark sample
+tests:
+  samples.bluetooth.iso_connected_benchmark:
+    build_only: true
+    build_on_all: true
```

Comment:
You should not be building on all, (build_on_all). Also, why build_only? Is this not supposed to run being a benchmark and a sample?